app/assets/javascripts/blob/upload_file.js.coffee

```diff
@@ -0,0 +1,35 @@
+Dropzone.options.blobFileDropzone = false;
+
+$(document).on "page:load", ->
+      $("#blobFileDropzone").dropzone()
+
+Dropzone.options.blobFileDropzone = {
+  autoProcessQueue: false,
+  clickable: true,
+  uploadMultiple: false,
+  paramName: "file",
+  parallelUploads: 1,
+  maxFiles: 1,
+  addRemoveLinks: true,
+  previewsContainer: '.dropzone-previews',
```

Comment:
Can you also make the "click to upload" more obvious? For example, this is another example of how someone uses Dropzone. Notice the border as well:

![image](https://cloud.githubusercontent.com/assets/963826/9837835/5027ba14-5a00-11e5-878b-3f8dfaf66475.png)